src/chat/components/bubble/CustomBubble.tsx

```diff
@@ -98,7 +109,11 @@ export const CustomBubble: React.FC<CustomBubbleProps> = ({
       default: {
         return (
           <View>
-            <Bubble {...bubbleMessage} />
+            <PreviewLink
```

Comment:
```suggestion
            <MessageWithPreviewLink
```
This naming is a little bit not clear, I suggest we change it to above suggestion or `DefaultMessage` | `DefaultBubble`

Reply:
ok I updated